graphql-dgs-subscriptions-websockets-autoconfigure/src/main/kotlin/com/netflix/graphql/dgs/subscriptions/websockets/DgsWebSocketConfigurationProperties.kt

```diff
@@ -25,14 +25,23 @@ import javax.annotation.PostConstruct
 @ConfigurationProperties(prefix = "dgs.graphql.websocket")
 @Suppress("ConfigurationProperties")
 data class DgsWebSocketConfigurationProperties(
-    @DefaultValue("/subscriptions") var path: String = "/subscriptions"
+    @DefaultValue("/subscriptions") var path: String = "/subscriptions",
+    /** Connection Initialization timeout in ms. */
+    @DefaultValue("10000") var connectionInitTimeout: String = "10000"
```

Comment:
Why not make this a `Duration`, Spring will handle changing the seconds (`s`), millis (`ms`), etc to the proper value.